app/components/appMainModule/committeePageModule/committeePageRoutes.js

```diff
@@ -8,19 +8,23 @@
 'use strict';
 
 function committeePageRoutes($stateProvider) {
-//module.exports = function($stateProvider) {
   $stateProvider.state({
     name: 'appMain.committee',
-    // name: 'appMain.locality.committee',
     abstract: true,
-    url: '^/committee/:committee_id',
+    url: '^/committee/:filer_id',
     template: '<ui-view></ui-view>',
     controller: 'committeePageController',
     resolve: {
-      committee: function($stateParams, disclosureApi) {
-        return disclosureApi.committee.get({
-          committee_id: $stateParams.committee_id
+      committee: function($stateParams, static_api) {
+        return static_api.committee.get({
+          filer_id: $stateParams.filer_id
+        });
+      },
+      contributions: function($stateParams, static_api) {
+        var apiCall = static_api.committee.contributions({
+          filer_id: $stateParams.filer_id
         });
+        return apiCall.$promise;
```

Comment:
Yeah, ngResource actually caused a number of problems here - doesn't return promises, plays badly with ag-grid. And I'm not sure it makes sense since some of the endpoints (like committee contributions) aren't very rest-y. I'd actually vote for scrapping it, but it's probably not worth the work to refactor it out.

Ag-grid can also be pointed directly at a datasource and take advantage of serverside pagination.